src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+spring.datasource.url = ${JDBC_DATASOURCE}
 
+# Username and password
+spring.datasource.username = ${JDBC_USERNAME}
+spring.datasource.password = ${JDBC_PASSWORD}
+
```

Comment:
Od razu pomyślałbym o dedykowanej bazie dla serwera Heroku, żeby w momencie merge, ta aplikacja nadal działała.

Czyli sprowadzi się to do ustawienia tych wartości na serwerze heroku, jak macie czystą bazkę gotową do podpięcia to podejdę i skonfigorujemy Heroku.

Flyway'a nie ma, ale w początkowej fazie create-drop będzie ok. Pamiętajcie tylko, że przy każdym merge, będzie wam czyścić bazę na Heroku.

Reply:
Heroku jest skonfigurowane, zaraz zrobione druga baze do pracy lokalnie